LeagueOfTowers/Assets/Scripts/CurrencyManager.cs

```diff
@@ -10,7 +10,7 @@
 public class CurrencyManager : Singleton<CurrencyManager>
 {
     //an event that is triggered when the currency changes
-    public event CurrencyChanged Changed;
+    //public event CurrencyChanged Changed;
 
```

Comment:
I may be wrong, but if these variables are not needed anymore, wouldn't it be better if we delete them?

Reply:
i removed.